testing/path/common.py

```diff
@@ -186,9 +188,10 @@ def test_visit_norecurse(self, path1):
         assert "sampledir" in l
         assert not path1.sep.join(["sampledir", "otherfile"]) in l
 
-    def test_visit_filterfunc_is_string(self, path1):
+    @pytest.mark.paramtrize('fil', ['*dir', u'*dir'])
```

Comment:
This needs to ensure its bytes in python3

For ensuring propper Funktion i propose also using a non-ascii string